readme.md

```diff
@@ -8,7 +8,11 @@
 ## Install
 
 ```
-$ hyper install hyper-snazzy
+hyper install hyper-snazzy
+```
+Note: You may need to run the below command first if you don't have Hyper CLI installed
+```
+sudo ln -sf "/Applications/Hyper.app/Contents/Resources/bin/hyper" "/usr/local/bin/hyper"
```

Comment:
This is not the place to document Hyper things. Link to some Hyper docs instead.

Reply:
> I couldn't find this info on https://hyper.is/ so I couldn't link it.

Sounds like you should open an issue on Hyper instead about that.